libs/core/core/components/use-alt-link-download-preventer-effect.hook.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { useCallback, useEffect } from "react"
+
+const useAltLinkDownloadPreventerEffect = () => {
+  const handleClick = useCallback((event: MouseEvent) => {
+    if (event.altKey && !event.target) {
```

Comment:
Looks suspicious. If `altKey === false` and parent id anchor than prevent default.

Reply:
Let's discuss on sync.